.github/workflows/ping-codeowners.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Ping code owners'
+on:
+  issues:
+    types: [labeled]
+
+jobs:
+  ping-owner:
+    if: ${{ contains(github.event.label.name, 'exporter/') || contains(github.event.label.name, 'extension/') || contains(github.event.label.name, 'processor/') || contains(github.event.label.name, 'receiver/') }}
```

Comment:
I think there are also code owners for `cmd/`, `internal/`, `pkg/`, and `tracegen/`.  Should we also include those?  I think at minimum `cmd/` and `pkg/` should be included.

Reply:
we'll need to add more labels for all the components, but it makes sense to me